CONTRIBUTING.md

```diff
@@ -1,28 +1,36 @@
 # How to Contribute
 
-Contributions here are more than welcomed! You can contribute to [rector-src](https://github.com/rectorphp/rector-src) repository.
+Contributions here are more than welcomed! You can contribute to [rector-src](https://github.com/rectorphp/rector-src) repository or one of [extension packages](https://github.com/rectorphp/).
 
 ## Preparing Local Environment
 
-If you have PHP 8 and Composer installed locally you can use it straight away. You can validate your environment with:
+1. Fork the [rector/rector-src](https://github.com/rectorphp/rector-src) repository and clone it
 
 ```bash
-composer check-platform-reqs
+git clone git@github.com:rectorphp/rector-src.git
```

Comment:
should clone from fork:

```suggestion
git clone git@github.com:your-username/rector-src.git
```

Reply:
Just noticed... will update

Reply:
How would you phrase it?
~90 % contributions goes here, the rest to vendor packages